localisation/replace/odessus_replace_l_english.yml

```diff
@@ -0,0 +1,3 @@
+﻿l_english:
+ timariot_system:0 "Timar System"
```

Comment:
in order not to confuse our future selves, we shouldnt put anything here we dont actually change